staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -150,42 +161,70 @@ func (a *AdmissionOptions) Validate() []error {
 
 	errs := []error{}
 
+	if len(a.PluginNames) > 0 &&
```

Comment:
would prefer comparison with nil here instead. An empty `PluginNames` (= no enabled plugins) is different from a nil value (= unconfigured).

Reply:
I see.